frontend/packages/knative-plugin/src/components/overview/__tests__/ConfigurationsOverviewListItem.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink } from '@console/internal/components/utils';
+import { ConfigurationModel } from '@console/knative-plugin';
+import ConfigurationsOverviewListItem from '../ConfigurationsOverviewListItem';
+import { mockConfigurationsData } from '../__mocks__/overview-knative-mock';
+
+describe('ConfigurationsOverviewListItem', () => {
+  it('should component exists', () => {
+    const wrapper = shallow(
+      <ConfigurationsOverviewListItem configuration={mockConfigurationsData[0]} />,
+    );
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should list the Configuration', () => {
+    const wrapper = shallow(
+      <ConfigurationsOverviewListItem configuration={mockConfigurationsData[0]} />,
+    );
+    expect(wrapper.find('li')).toHaveLength(1);
+    expect(
+      wrapper
+        .find('li')
+        .at(0)
+        .props().className,
+    ).toEqual('list-group-item');
+  });
+
+  it('should have ResourceLink with proper kind', () => {
+    const wrapper = shallow(
+      <ConfigurationsOverviewListItem configuration={mockConfigurationsData[0]} />,
+    );
+    expect(wrapper.find(ResourceLink)).toHaveLength(1);
+    expect(
+      wrapper
+        .find(ResourceLink)
+        .at(0)
+        .props().kind,
+    ).toEqual(referenceForModel(ConfigurationModel));
+  });
+
+  it('should display latestCreatedRevisionName and latestReadyRevisionName', () => {
+    const wrapper = shallow(
+      <ConfigurationsOverviewListItem configuration={mockConfigurationsData[0]} />,
+    );
+    expect(wrapper.find('span')).toHaveLength(4);
```

Comment:
Let's not assert presence of the markup like this because we're interested in the data and not so much the markup.